hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp

```diff
@@ -22,6 +22,6 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Cool. Testing looks good but I want to run another iteration without all the debug output I have sprinkled throughout the Python script. AND, there is one last (I hope) issue we need a decision on (it's a matter of preference I suppose) ... the Python script (*UpdateCopyrightHeader.py*) is called, from *UpdateDirCopyrightHeaders.sh* which is called from *UpdateAllCopyrightHeaders.sh*. The Python script now has a new argument (*--update-mode*) - a value of 1 performs the copyright header updates, a value of 0 outputs what files need copyright header updating (it makes NO changes). Here is my question - do we want to propagate the *--update-mode* parameter down from the first shell script to the second? **OR** do we simply have people edit the second script (*UpdateDirCopyrightHeaders.sh*) at the line where *UpdateCopyrightHeader.py* is called? The change I am going to commit will have *--update-mode=0* (default of no updating, just checking). Maybe it's not a big deal but I thought I'd ask anyhow.